### PR TITLE
Add docs for CLI orphan fragments

### PR DESCRIPTION
**Pull request by SmileyChris**
`towncrier create +.feature.rst` wasn't documented.

### CODE REVIEW

File: docs/cli.rst
Comment (adiroiban) [inline docs/cli.rst]: Maybe use [semantic newlines](https://sembr.org/) for the documentation

I think that this is a good place to remind what "orphan fragments" are :)

Minor suggestion

```suggestion
To create a news fragment not tied to a specific issue,
what is called an orphan fragment,
start the fragment name with a ``+``.
If that is the entire fragment name,
a random hash will be added for you::
```

**Review by adiroiban — APPROVED** [review]
Thanks. This is much better. Much appreciated :)

I was always confused by the orphan thing...as I don't use or need it

I think is best to also mention this behaviour in the create docstring

https://github.com/twisted/towncrier/blob/914b446e64d62f3968a97931e490b1abe5c8c581/src/towncrier/create.py#L46-L68

In this way, the info is found in `towncrier create --help`

The idea is that in the future we can just run some helper utility for click and it will automatically generate the .RST documentation for the CLI.

I think is best to keep as much info as possible into the `--help` part.

For the `--help` docstring , we can keep the info brief,
Just mention that you can use `+` as the ticket number and towncrier will auto-generate

More info and examples, can be found in the RST version.

Thanks again

File: docs/cli.rst
Comment (SmileyChris) [reply docs/cli.rst]: Done!

**Comment by adiroiban** [comment]
Thanks Chris. I have merged this. I added you to the list of maintainer, so if the future you should be able to merge a PR once it is approved.

Just make sure that any change is reviewed and approved before a merge.

Thanks again!

**Pull request merged**
